src/components/Buttons/IconButton/IconButton.jsx

```diff
@@ -0,0 +1,18 @@
+import PropTypes from "prop-types";
+import "src/components/Buttons/buttons.scss";
+
+function IconButton({ buttonSrc, buttonAltText, onAction }) {
+  return (
+    <button onClick={onAction} className="button-icon">
```

Comment:
The component name should reflect its purpose and functionality, considering its usage beyond just an icon.

Reply:
@rsa-wtag can you explain it a bit more maybe with some examples or suggestions?

Reply:
@nsi-wtag The component is rendering not only icon but the whole button with its functionalities. So the component name `Button` would be more relevant.

Reply:
@rsa-wtag vai, as the primary purpose of this component is to render buttons those are directly icons (according to design) instead of texts, I think the name `IconButton` sounds more direct and explicit about the purpose of the component rather than using a generic name like `Button`.